### DISCUSSION TITLE
Stanza 1.1.1 can no longer be auto downloaded

### DISCUSSION THREAD

**Discussion by argosopentech** [argosopentech/argos-translate#74] [General]
https://github.com/stanfordnlp/stanza/issues/667

**Comment by PJ-Finlay** [comment]
Apparently it's an issue with their file servers so hopefully not a long term issue.

**Reply by pierotofy** [reply]
Wonder if there's a way to setup a mirror.

**Reply by PJ-Finlay** [reply]
Potentially, the Stanza models are packaged as files so this only effects making new packages not existing ones. Also I'm hoping to [do sentence boundary detection with the seq2seq model](https://github.com/argosopentech/onmt-models/tree/132f3752589a0ac9cb8b2a0af00d7988cda87fbb/sbd) in future versions.

**Comment by PJ-Finlay** [comment]
Back up: https://github.com/stanfordnlp/stanza/issues/667#issuecomment-815033200
